pyproject.toml

```diff
@@ -11,6 +11,7 @@ PyYAML = "^6.0"
 invenio-search = {version = "^1.4.2", extras = ["elasticsearch7"]}
 jsonpatch = "^1.32"
 invenio-db = "^1.0.14"
+invenio-records-rest = "*"
```

Comment:
`invenio-records-rest = '>=1.8.0'` ?

Reply:
We won't place any restriction into `rero-invenio-base` for this package because it seems unnecessary. Client project including `rero-invenio-base` could define a restriction if necessary.